SCClassLibrary/Common/Files/ExampleFiles.sc

```diff
@@ -10,4 +10,8 @@ ExampleFiles {
 	*child {
 		^Platform.resourceDir +/+ "sounds" +/+ "a11wlk01.wav";
 	}
+
+	*folder {
```

Comment:
This is  mixing unix (directory) and windows (folder) vocabulary - I think within sclang we tend more towards dir, see e.g. `Platform.resourceDir`).